webapp/app/js/perf.js

```diff
@@ -28,11 +28,15 @@ perf.factory('seriesSummary', ['seriesColors', function(seriesColors) {
     if (signatureProps.job_group_symbol === "T-e10s") {
       extra = " e10s";
     }
-    var testName = signatureProps.suite + " " + signatureProps.test +
+    var testName = signatureProps.test;
+    if (testName === undefined)
+      testName = "summary";
+
+    var name = signatureProps.suite + " " + testName +
       " " + optionCollectionMap[signatureProps.option_collection_hash] + extra;
-    var signatureName =  testName;
-    return { name: signatureName, signature: signature, platform: platform,
-             testName: testName, projectName: projectName, color: seriesColors[number] };
+    var signatureName = name;
+    return { name: name, signature: signature, platform: platform,
+             projectName: projectName, color: seriesColors[number] };
```

Comment:
here we define signatureName but don't use it.  Previously in the return statement we had:
"name: signatureName" as well as "testName: testName"

I understand that name is what testName used to be, so lets just remove signatureName.